packages/account/src/Sections/Security/TwoFactorAuthentication/two-factor-disabled.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import QRCodeSVG from 'qrcode.react';
+import { MobileWrapper, ThemedScrollbars, Text, Timeline, Loading, Clipboard, DesktopWrapper } from '@deriv/components';
+import TwoFactorAuthenticationArticle from './two-factor-authentication-article';
+import { Localize, localize } from '@deriv/translations';
+import DigitForm from './digit-form';
+import { useStore } from '@deriv/stores';
+
+type TTwoFactorDisabled = {
+    secret_key: string;
+    qr_secret_key: string;
+    is_qr_loading: boolean;
+};
+
+const TwoFactorDisabled = ({ secret_key, qr_secret_key, is_qr_loading }: TTwoFactorDisabled) => {
+    const { ui } = useStore();
+    const { is_mobile } = ui;
+    return (
+        <React.Fragment>
+            <ThemedScrollbars is_bypassed={is_mobile} autohide className='two-factor__scrollbars'>
+                <MobileWrapper>
+                    <TwoFactorAuthenticationArticle />
+                </MobileWrapper>
```

Comment:
```suggestion
                { is_mobile && <TwoFactorAuthenticationArticle />
```
maybe we could go with this implementation?
same thing for DesktopWrapper down in the code :)

Reply:
I think this works fine.